File: 2AP4_2/python_1/suma.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma.py

def main(args):
    suma = 0
    i = 0
    n = int(input("Ile podasz liczb? "))
    for i in range(n):
        a = int(input("Podaj liczbę: "))
        suma = suma + a
    print(suma)
        
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: 2AP4_2/cpp_1/suma.cpp
/*
 * suma.cpp
 * 
 */
#include <iostream>

using namespace std;

int main(int argc, char **argv) {
    int suma = 0;
    int i = 0;
    int n = 0;
    int a = 0;
    cout << "Ile podasz liczb? ";
    cin >> n;
    cout << "Podaj liczbę: ";
    for (i = 0; i < n; i = i + 1) {
        cin >> a;
        suma = suma + a;
    }
    cout << suma;

    return 0;
}

